jstz_cli/src/repl.rs

```diff
@@ -0,0 +1,119 @@
+use boa_engine::Source;
+use jstz_core::runtime;
+use jstz_core::runtime::Runtime;
+use rustyline::error::ReadlineError;
+use rustyline::Editor;
+use tezos_smart_rollup_mock::MockHost;
+
+use jstz_core::host::HostRuntime;
+
+use jstz_api::http::HttpApi;
+use jstz_api::url::UrlApi;
+use jstz_api::ConsoleApi;
+use jstz_api::KvApi;
+use jstz_api::TextEncoderApi;
+
+use jstz_proto::api::ContractApi;
+use jstz_proto::api::LedgerApi;
+
+use jstz_crypto::public_key_hash::PublicKeyHash;
+
+use anyhow::Result;
+
+pub fn exec(self_address: Option<String>) -> Result<()> {
+    let mock_address_string = "tz4FENGt5zkiGaHPm1ya4MgLomgkL1k7Dy7q";
```

Comment:
Might be good to use a more obvious mock address. I think `tz4RepLRepLRepLRepLRepLRepLRepN7Cu8j` should work nicely